docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,85 @@
+  ⮕ `Administrator::edit(AdministratorData $administratorData, EncoderFactoryInterface $encoderFactory, self $administratorByUserName = null)`
+- `AdministratorService::delete(Administrator $administrator, $adminCountExcludingSuperadmin)`  
+  ⮕ `Administrator::checkCanBeDeleted(TokenStorageInterface $tokenStorage, int $adminCountExcludingSuperadmin)`
+- `OrderService::calculateTotalPrice(Order $order)`  
+  ⮕ `Order::calculateTotalPrice(OrderPriceCalculation $orderPriceCalculation)`
+- `ProductService::getProductSellingPricesIndexedByDomainIdAndPricingGroupId(Product $product, array $pricingGroups)`  
+  ⮕ `ProductFacade::getAllProductSellingPricesIndexedByDomainId(Product $product)`
+- `ProductService::sortProductsByProductIds(array $products, array $orderedProductIds)`  
+  ⮕ `ProductRepository::getSortedProductsByIds($domainId, PricingGroup $pricingGroup, array $sortedProductIds)`
+- `ProductService::markProductForVisibilityRecalculation(Product $product)`  
+  ⮕ `Product::markForVisibilityRecalculation()`
+- `ProductService::edit(Product $product, ProductData $productData)`  
+  ⮕ `Product::edit(ProductCategoryDomainFactoryInterface $productCategoryDomainFactory, ProductData $productData, ProductPriceRecalculationScheduler $productPriceRecalculationScheduler)`
+- `ProductService::delete(Product $product)`  
+  ⮕ `Product::getProductDeleteResult()`
+- `ProductService::recalculateInputPriceForNewVatPercent(Product $product, $productManualInputPrices, $newVatPercent)`  
+  ⮕ `ProductManualInputPrice::recalculateInputPriceForNewVatPercent($inputPriceType, BasePriceCalculation $basePriceCalculation, InputPriceCalculation $inputPriceCalculation, $newVatPercent)`
+- `OrderService::getOrderDetailUrl(Order $order)`  
+  ⮕ `OrderUrlGenerator::getOrderDetailUrl(Order $order)`
+- `OrderService:: createOrderProductInOrder(Order $order, Product $product, Price $productPrice)`  
+  ⮕ `Order::createOrderProduct(Product $product, Price $productPrice, OrderProductFactoryInterface $orderProductFactory, Domain $domain, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderService::editOrder(Order $order, OrderData $orderData)`  
+  ⮕ `Order::edit(OrderData $orderData, OrderItemPriceCalculation $orderItemPriceCalculation, OrderProductFactoryInterface $orderProductFactory, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderStatusService::checkForDelete`  
+  ⮕ `OrderStatus::checkForDelete`
+
+Following classes have been removed:
+- `AdministratorService`
+- `AdministratorGridService`
+- `CustomerService`
+- `ProductService`
+- `OrderService`
+- `OrderStatusService`
+- `FlagService`
+
+Following methods have been removed:
+- `User::setDeliveryAddress()`, use `User::editDeliveryAddress()` instead
+- `Administrator::addGridLimit()`, use `Administrator::rememberGridLimit()` instead
+- `Administrator::removeGridLimit()`
```

Comment:
for those two, I would add "...as it was not used anywhere" - so I am aware that there is no alternative. Now, it seems like you just forget to put there the alternative :slightly_smiling_face: